plugins/nginx-vhosts/functions

```diff
@@ -8,18 +8,22 @@ source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
 get_nginx_location() {
   declare desc="check that nginx is at the expected location and return it"
-  local NGINX_LOCATION="/usr/sbin/nginx"
-  if ! [ -x "$NGINX_LOCATION" ]; then
-    dokku_log_fail "Could not find nginx binary at ${NGINX_LOCATION}."
-    exit 1;
+  local NGINX_LOCATION=$(which nginx 2>/dev/null)
+
+  if [[ -z "$NGINX_LOCATION" ]]; then
+    NGINX_LOCATION="/usr/sbin/nginx"
+    if ! [[ -x "$NGINX_LOCATION" ]]; then
```

Comment:
You might want to move this out of the containing if statement, and also turn it into:

```
if [[ ! -x "$NGINX_LOCATION" ]]; then
```

Reply:
Done and done.